notebooks/parler-tts-text-to-speech/parler-tts-text-to-speech.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    import soundfile as sf
</pre><p>I did not see soundfile in installing requirements. Should it be added?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2114/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>